### ISSUE TITLE
tensorflow-1-public/C1/W2/assignment/C1_W2_Assignment_Solution.ipynb fix?

### ISSUE THREAD

**Issue by gladwig2** [https-deeplearning-ai/tensorflow-1-public#13]
![image](https://user-images.githubusercontent.com/8951837/129588716-eadf3f61-5571-4ddc-aee1-5a553b351949.png)

I don't understand this 'fix'

In my experiments, logs.get('accuracy') _Always_ returns **None**.

so you will never exit the callback due to achieving a target accuracy.?

As noted in the previous issue, a fix would be to use 'loss' which does return a real value. Note, the grader may need to be updated as well.
[Edit]
I see that logs.get('accuracy') **Does** run successfully in the **CoLab environment**.

So, I guess this  issue should really be to update the Coursera environment?

Example: In Coursea, Exercise 2, I have:
`    class myCallback(tf.keras.callbacks.Callback):
      def on_epoch_end(self, epoch, logs={}):
        print(f"accuracy: {logs.get('accuracy')}")
        if(logs.get('loss')<0.01):
          print("\nReached 99% accuracy so cancelling training!")
          self.model.stop_training = True
`

I get output like this: 
`Epoch 9/9
59424/60000 [============================>.] - ETA: 0s - loss: 0.0126 - acc: 0.9957accuracy: None
60000/60000 [==============================] - 10s 162us/sample - loss: 0.0126 - acc: 0.9957`

logs.get('accuracy') is returning **None.** so I used logs.get('loss') instead.

I guess I am not really sure what's going on. Is there a compile/other flag that allows logs.get('accuracy') to work on Coursera?

The difference between the co-lab and coursera environments are confusing. Why not do it all on Coursera?

Also, curiously, the 
 model.fit()
output includes accuracy in its printout.
